week08_pomdp/practice_pytorch.ipynb

```diff
@@ -6,11 +6,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import os\n",
     "import sys\n",
     "if 'google.colab' in sys.modules:\n",
     "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/atari_util.py\n",
     "    !wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/0ccb0673965dd650d9b284e1ec90c2bfd82c8a94/week08_pomdp/env_pool.py\n",
-    "\n",
+    "    # Setup the attari driver for video games\n",
+    "    !wget -q https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/setup_colab.sh -O- | bash\n",
+    "    !touch .setup_complete\n",
+    "    # Install packages dependencies\n",
+    "    os.system(\"pip install -q scipy==1.1.0\")\n",
```

Comment:
Are you sure it's a good idea to downgrade `scipy`? The current version in Colab is 1.7.3; 1.1.0 was [released](https://github.com/scipy/scipy/releases/tag/v1.1.0) on 2018-05-05, more than 4 years ago.

What are you fixing with this downgrade?

Reply:
Because it gives an error inside your module code ` PreprocessAtari` on Colab which has a missing module called `imresize.scipy.misc`; 

![image](https://user-images.githubusercontent.com/72295771/201531368-2163f50c-bb9a-43a4-884d-6598e727f49d.png)

Reply:
It seems we fixed that issue with `imresize` a long time ago ([here](https://github.com/yandexdataschool/Practical_RL/commit/7404c2d0758a0f3da88e9dce10e4a3202d378620#diff-26b48625335d4d26051e0c4d2d6f56fb01db57243707365024e185642aa6b9d2L5), in April 2020), but forgot to update the links to `atari_util.py`. So instead of downgrading `scipy` it would be better to update the commit hashes in the links.

Reply:
okey, let me do that for now!